templates/dart/README.md.twig

```diff
@@ -35,7 +35,7 @@ dependencies:
 You can install packages from the command line:
 
 ```bash
-pub get {{ language.params.packageName }}
+pub add {{ language.params.packageName }}
```

Comment:
This won't actually run successfully. It should be:

```suggestion
dart pub add {{ language.params.packageName }}
```

Reply:
Fixed